backend-project/small_eod/cases/tests.py

```diff
@@ -30,31 +41,52 @@ def test_tag_field(self):
     def test_raise_for_over_maximum_feature(self):
         dictionary = DictionaryFactory(max_items=3)
         features = FeatureFactory.create_batch(size=5, dictionary=dictionary)
-        serializer = CaseCountSerializer(data={
-            "name": "Polska Fundacja Narodowa o rejestr umów",
-            "audited_institution": [],
-            "comment": "xxx",
-            "responsible_user": [],
-            "notified_user": [],
-            "feature": [x.id for x in features],
-            "tag": [],
-        })
+        serializer = CaseCountSerializer(
+            data={
+                **self.default_data,
+                **{"feature": [x.id for x in features], "tag": [],},
+            },
+            context={"request": self.request},
+        )
         self.assertFalse(serializer.is_valid())
-        self.assertEqual(set(serializer.errors.keys()), {'feature'})
+        self.assertEqual(set(serializer.errors.keys()), {"feature"})
 
     def test_serializer_counters(self):
-        CaseFactory()
+        note = NoteFactory()
         case_counted = Case.objects.with_counter().get()
         self.assertEqual(case_counted.letter_count, 0)
-        self.assertEqual(case_counted.note_count, 0)
+        self.assertEqual(case_counted.note_count, 1)
         data = CaseCountSerializer(case_counted).data
         self.assertEqual(data["letter_count"], 0)
-        self.assertEqual(data["note_count"], 0)
+        self.assertEqual(data["note_count"], 1)
+
+    def test_default_for_related_user(self):
+        del self.default_data["responsible_user"]
```

Comment:
Np
````python
data = self.get_serializer_data(responsible_user=[], notified_user=[])
````

Reply:
Ale w tym wypadku te wartości będą null

Reply:
Dostrzegam, że pusta tablica, a pominięcie pola to różne przypadki, dlatego użyłem "del". Można wprowadzić nazwane pole, ale nie chciałem wyciągać abstrakcji na pierwszych przypadkach, bo zwykle mała refaktoryzacja po 3-4 przypadku jest sprawniejsza, skuteczniejsza i pozwala uchronić przed przedinżynierowaniem.